### DISCUSSION TITLE
Build number in changelog for cosmetic/non-code related changes

### DISCUSSION THREAD

**Discussion by rohitab** [FarGroup/FarManager#621] [General]
**Information**
As described below, changes do not always require a build increase. In these cases, should the entry in the changelog file use the previous build number, or omit it?

https://github.com/FarGroup/FarManager/blob/42b1fb183cae0f74e678d7499ee91cec1907e675/CONTRIBUTING.md?plain=1#L54-L59

If the build number is omitted, as shown in the first entry below, the `ChangelogHeader.lua` macro generates the new build number using the time zone offset. In this case, it generated a build number of `801` instead of `6098`.

**changelog from https://github.com/FarGroup/FarManager/commit/9ec1f71cc88e72269905fb6fcf1a5934cafeeb7a**
```
--------------------------------------------------------------------------------
MZK 28.01.2023 18:41:14 -0800

1. gh-618: Adjusted hotkey assignments on Search / Replace and Find File dialogs.

--------------------------------------------------------------------------------
drkns 24.01.2023 20:29:40 +0000 - build 6097

1. Continue 6095.

--------------------------------------------------------------------------------
```

**Question**
What should be the correct entry here? 1 or 2?

1. `MZK 28.01.2023 18:41:14 -0800`
2. `MZK 28.01.2023 18:41:14 -0800 - build 6097`

**Comment by alabuzhev** [comment]
We usually increment the build number when there are changes in the code, mostly to have a point of reference when something goes wrong. When changes are not it the code, it is way less likely that something can go wrong and users will have to report it, so we don't need a point of reference that much and the old build number will do. It does not affect anything else anywhere else.

ChangelogHeader.lua simply does not cover such case and prints rubbish. It would be nice to fix it, but such cases are so rare that no one could be bothered.

The first entry is correct.

**Comment by rohitab** [comment]
Thank you. Just wanted to make sure so I don't mess up your changelogs.
